src/previewgenerator.cpp

```diff
@@ -38,13 +38,17 @@ void PreviewGenerator::registerPlugin(MOBase::IPluginPreview *plugin)
 
 bool PreviewGenerator::previewSupported(const QString &fileExtension) const
 {
-  return m_PreviewPlugins.find(fileExtension.toLower()) != m_PreviewPlugins.end();
+  auto it = m_PreviewPlugins.find(fileExtension.toLower());
+  if (it == m_PreviewPlugins.end()) {
+    return false;
+  }
+  return it->second->isActive();
 }

```

Comment:
If we have two preview plugins that support the same file extension and the first found one is inactive, wouldn't the second be skipped as well?

Reply:
Well, since `m_PreviewPlugins` is not a multimap, you cannot have two plugins registered for the same file extension currently as far as I know.